Lib/pathlib.py

```diff
@@ -749,7 +749,10 @@ def __ge__(self, other):
 
     root = property(attrgetter('_root'),
                     doc="""The root of the path, if any.""")
-
+    
+    str = property(methodcaller('__str__', 
```

Comment:
methodcaller is redundant here, remove it (and it's import) and just pass __str__ directly as it's function signature matches the one required by property.